sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,15 @@
+-- Greengill Slave
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '' WHERE `entry` = 25084;
+
+-- These are reordered from c++-script so that conditions will work on linked events properly.
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 25084);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(25084, 0, 0, 1, 8, 0, 100, 0, 45109, 0, 0, 0, 11, 45111, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Greengill Slave - On Spellhit \'Orb of Murloc Control\' - Cast \'Enrage\''),
+(25084, 0, 1, 2, 61, 0, 100, 0, 0, 0, 0, 0, 123, 100000, 0, 0, 0, 0, 0, 19, 25060, 70, 0, 0, 0, 0, 0, 'Greengill Slave - On Spellhit \'Orb of Murloc Control\' - Add 100000 Threat to nearest Darkspine Myrmidon'),
+(25084, 0, 2, 3, 61, 0, 100, 0, 0, 0, 0, 0, 49, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Greengill Slave - On Spellhit \'Orb of Murloc Control\' - Start Attacking'),
```

Comment:
The npc will not attack its self.

Reply:
I tried using nearest of 25060 within 70 yards and it didn't seem to work, maybe removing the threat line will affect that